README.md

```diff
@@ -1,87 +1,196 @@
-  - `md-utils`: utilities for parsing markdown files
-- tutorials:
-  - `<tutorial-name>`
-    - `/builderdao.config.js`: configuration for the tutorial
-    - `/content/<md-file>`: tutorial content
-- programs:
-  - `<solana-program-name>`: a solana program
-    - `/src`: rust source code for the program
-    - `/ts`: SDK source code
-    - `/tests/`: rust test code for the program
-    - `/__tests__`: typescript tests for the program
-
-### Build
-
-To build all apps and packages, run the following command:
+This repo contains the [Kafé learning platform](https://dev.builderdao.io) including the web frontend, Solana programs and TypeScript SDK.
 
-```
-yarn run build
-```
+Click each heading below for details:
 
-### Develop
+<details>
+ <summary>/apps</summary>
 
-To develop all apps and packages, run the following command:
+- `/web` - Kafé by Builder DAO frontend
+  - `/components` - ReactJS components
+  - `/hooks` - React Hooks specific to Kafé
+  - `/pages` - Next.js API & page routes
+  - `/styles` - This project uses [TailwindCSS](https://tailwindcss.com/)
+  </details>
 
-```
-yarn run dev
-```
+<details>
+ <summary>/dao-programs</summary>
+
+- `/migrations/deploy.js` - Deployment script
+- `/programs/<solana-program-name>` - Solana program
+  - `/src/`: Rust source code for the program
+  - `/tests/`: Rust tests for the program
+  - `/ts-sdk/`: TypeScript SDK source code
+  </details>
+
+<details>
+ <summary>/packages</summary>
+
+- `/ui`- A stub React component library
+- `/config` - `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `/tsconfig` - TypeScript configuration used throughout the repo
+- `/md-utils` - Utilities for parsing markdown files
+</details>
+
+<details>
+ <summary>/templates</summary>
+
+- `/multipage` - A template for longer guides
+- `/simple` - A single page guide template
+</details>
+
+<details>
+ <summary>/token-scripts</summary>
+
+- `/keypairs` - Solana keypairs for token scripts
+- `/token-list` - SPL token metadata directory
+- `/token-list/logo/` - SPL token images directory
+- `/token-list/token-registry.json` - SPL token metadata
+- `/utils` - TypeScript SDK helpers
+</details>
+
+<details>
+ <summary>/tutorials</summary>
+
+- `/<guide-name>`
+  - `/content/assets/` - Images used in the guide are stored here
+  - `/content/index.mdx` - Main guide content
+  - `/content/<filename>.mdx` - Additional pages, if required
+  - `/builderdao.config.json` - Configuration file for the guide
+  - `/builderdao.lock.json` - Autogenerated guide lockfile
+  </details>
+
+---
 
 ## How to use?
 
-### Quick start for authors.
+This repo is structured as a monorepo using [turborepo](https://turborepo.org), which keeps things modular and gives us the ability to build on top of the Builder DAO SDK independent of Kafé. We use [Yarn](https://yarnpkg.com/) as a package manager. See the repo contents in the previous section to understand how the repo is structured.
```

Comment:
Let's add a link to monorepo ---> monorepo.tools
Don't need the last sentence ("See the repo...")